.travis.yml

```diff
@@ -43,9 +43,17 @@ script:
   - if [[ $LINT_TEST ]]; then composer run-script lint; fi
 
 before_deploy:
-  - if [[ ! -f ./release/upgrade-code.phar ]]; then php build.php; fi
-  - mkdir -p ./release
-  - if [[ -f ./upgrade-code.phar ]]; then mv upgrade-code.phar ./release/; fi
+  - if [ -d "release" ]; then rm -rf release ; fi
+  - git clone --single-branch --branch gh-pages https://github.com/$TRAVIS_REPO_SLUG.git release
+  - echo "UPGRADER_VERSION=$TRAVIS_TAG" > .env
```

Comment:
Will `.env` ever not have a newline at the end of the file?

Reply:
The `.env` file will be built into the PHAR file. Once the PHAR file is published, we don't keep it around. So that .env file will be regenerated from scratch every time.